PeerioTechnologies/peerio-desktop
# chore: upgrade prosemirror deps

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/2208/update-prosemirror-related-dependencies

#### Testing instructions  

All package upgrades were minor so no breaking changes are expected -- except that the upgrade did create a minor breaking change in the schema. Should be backwards-compatible regardless, but viewing/sending rich text messages should be re-tested a bit just in case.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
